tests/test_forcedPhot.py

```diff
@@ -79,6 +146,100 @@ def testRun(self):
         self.assertFloatsNotEqual(measCat["base_TransformedCentroid_x"], self.refCat['truth_x'])
         self.assertFloatsNotEqual(measCat["base_TransformedCentroid_y"], self.refCat['truth_y'])
 
+    def testRunQuantum(self):
+        config = ForcedPhotCcdTask.ConfigClass()
+        config.useVisitSummary = False
+        config.doApplySkyCorr = False
+        config.idGenerator.packer.name = "observation"
+        config.idGenerator.packer["observation"].n_detectors = 5
+        config.idGenerator.packer["observation"].n_observations = 10
+        pipeline_graph = PipelineGraph(universe=self.universe)
+        pipeline_graph.add_task("forcedPhotCcd", ForcedPhotCcdTask, config)
+        pipeline_graph.resolve(dimensions=self.universe)
+        init_outputs = []
+        (task,) = pipeline_graph.instantiate_tasks(
+            get_init_input=lambda _: SourceCatalog(self.refCat.schema),
+            init_outputs=init_outputs,
+        )
+        ((output_schema_cat, _),) = init_outputs
```

Comment:
I don't know if this is any nicer to read than `init_outputs[0][0]`, though I grant that you could unpack multiple vars this way.